plugin.go

```diff
@@ -2,26 +2,26 @@ package se2
 
 import "fmt"
 
-type Module struct {
+type Plugin struct {
 	Environment string
 	Tenant      string
 	Namespace   string
 	Name        string
 }
 
-// NewModule instantiates a local v1.0.0 Module that can be used for various calls with compute.Client.
-// Note: this constructor alone does not perform any actions on a remote Compute instance.
-func NewModule(environment, tenant, namespace, name string) *Module {
-	module := &Module{
+// NewPlugin instantiates a local v1.0.0 Plugin that can be used for various calls with se2.Client.
```

Comment:
```suggestion
// NewPlugin instantiates a local v1.0.0 plugin that can be used for various calls with se2.Client.
```
Nit based on feedback in other repos.